src/cleanfigure.m

```diff
@@ -939,6 +939,30 @@ function removeNaNs(meta, handle)
     topRight    = [xLim(2); yLim(2)];
 end
 % =========================================================================
+function data = limitPrecision(data)
+  % Limit the precision of the given data
+  
+  % Only do something if the data is not empty
+  if isempty(data) || all(~isfinite(data))
+      return
+  end
+  
+  % Only consider finite values
+  mask = isfinite(data);
+
+  % Get the maximal value of the data
+  range = max(abs(data(mask)));
```

Comment:
Can we rename `range` into something else, e.g. `amax`, since by definition `range = max-min`?